### PR TITLE
Correct exit code if ``failures % 256 == 0`` due to UNIX semantics

### PR DESCRIPTION
**Pull request by lamby**
Signed-off-by: Chris Lamb <chris@chris-lamb.co.uk>

### CODE REVIEW

File: blockfindertest.py [hunk 1]
```diff
@@ -145,4 +145,4 @@ def test_capitalization(self):
         results = test_runner.run(test_suite)
         failures += len(results.errors)
         failures += len(results.failures)
-    sys.exit(failures)
+    sys.exit(bool(failures))
```
Comment (d1b) [inline blockfindertest.py]: I'd prefer to see this implemented like:
```
return_code = 0
if failures:
    return_code = 1
sys.exit(return_code)
```

Comment (lamby) [inline blockfindertest.py]: We could compromise on ``sys.exit(1 if failures else 0)`` :)

Comment (d1b) [inline blockfindertest.py]: @lamby sure :-)

**Comment by d1b** [comment]
@lamby thank you for your contribution. I merged #93 in which I committed as you to give you credit for your changes.

**Pull request closed without merging**